Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/MousePointer.cs

```diff
@@ -75,15 +84,33 @@ public override IMixedRealityController Controller
         /// <inheritdoc />
         public override void OnPreSceneQuery()
         {
+            // screenspace to ray conversion
             transform.position = CameraCache.Main.transform.position;
 
-            Ray ray = new Ray(Position, Rotation * Vector3.forward);
+            Ray ray = new Ray(transform.position, transform.rotation * Vector3.forward);
             Rays[0].CopyRay(ray, PointerExtent);
 
             if (MixedRealityRaycaster.DebugEnabled)
             {
                 Debug.DrawRay(ray.origin, ray.direction * PointerExtent, Color.green);
             }
+
+            // ray to worldspace conversion
+            Camera RaycastCamera = MixedRealityToolkit.InputSystem.FocusProvider.UIRaycastCamera;
+            RaycastCamera.transform.position = Rays[0].Origin;
+            RaycastCamera.transform.rotation = Quaternion.LookRotation(Rays[0].Direction);
+
+            // The RayCastCamera is placed so that the current cursor position is in the center of the camera's view space.
+            Vector3 viewportPos = new Vector3(0.5f, 0.5f, 1.0f);
+            cachedWorldSpacePosition = RaycastCamera.ViewportToWorldPoint(viewportPos);
```

Comment:
>cachedWorldSpacePosition [](start = 12, length = 24)

The mouse pointer is a controller pose synchronizer, meaning that it should synchronize the transform its game object to its own position.

Reply:
Can you give me more info on this? I'm not quite sure I understand what I'm supposed to do here?